polymer.json

```diff
@@ -7,5 +7,14 @@
     "src/shop-cart.html",
     "src/shop-checkout.html",
     "src/lazy-resources.html"
+  ],
+  "sourceGlobs": [
+   "src/**/*",
+   "images/**/*",
+   "bower.json"
+  ],
+  "includeDependencies": [
+    "manifest.json",
+    "bower_components/webcomponentsjs/webcomponents-lite.min.js"
```

Comment:
As for 'data/' yes, if you need it in the build and it's not picked up by the analyzer via HTML imports/loading then you should include it here